### DISCUSSION TITLE
how often to meet during the summer

### DISCUSSION THREAD

**Discussion by pdurbin** [bostonopen/bostonopen.github.io#5] [General]
At the May meetup there was a little discussion about the summer months.

Should we meet every month, like normal?

Every other?

@holmesworcester and I already picked a date for June (Sunday the 11th) so we don't plan to skip the summer entirely. However, July and August might be tough.

(By the way, all are very welcome to join [Quiet](https://tryquiet.org) where the dates for the next meetup are discussed. Please just get in touch with me or Holmes for an invite code.)

It feels like we compete a bit with good weather! ☀️ 😎 🏖️  At the May meetup, when it was gorgeous outside, we had 7 instead of 11. Here's the data:

    2023-05 7 people
    2023-04 11 people
    2023-03 11 people
    2023-02 Skipped!
    2023-01 11 people
    2022-12 10 people

What do YOU think? Especially when it comes to July and August?

**Comment by holmesworcester** [comment]
I'll be gone at the beginning of July but I'll be around in August I think! Summer Sunday evenings are a bit hard to give up though, so I'd be open to skipping July and August.

**Comment by pdurbin** [comment]
The summer hiatus seemed to work well. We had dipped to just 3 attendees in June but yesterday we were back to a healthy 10 people:

- 2023-09 10 people
- 2023-08 Skipped! Summer vacation! 🏖️
- 2023-07 Skipped! Summer vacation! 🏖️
- 2023-06 3 people
- 2023-05 7 people
- 2023-04 11 people
- [2023-03](http://blog.greptilian.com/2023/02/17/open-source-meetup-in-brookline-3/) 11 people
- 2023-02 Skipped!
- [2023-01](http://blog.greptilian.com/2023/01/03/open-source-meetup-in-brookline-2/) 11 people
- [2022-12](http://blog.greptilian.com/2022/12/10/open-source-meetup-in-brookline/) 10 people

(These numbers are from the bottom of https://bostonopen.dev by the way.)

**Comment by pdurbin** [comment]
Summer is about to roll around again. @orblivion and I are starting to talk about it here: https://osdc.zulipchat.com/#narrow/stream/406743-boston/topic/2024-06.20meetup/near/437243107
